patches/minecraft/net/minecraft/entity/boss/EntityDragon.java.patch

```diff
@@ -16,3 +16,9 @@
                          {
                              if (block != Blocks.field_150483_bI && block != Blocks.field_185776_dc && block != Blocks.field_185777_dd && block != Blocks.field_150411_aY && block != Blocks.field_185775_db)
                              {
+-                                flag1 = this.field_70170_p.func_175698_g(blockpos) || flag1;
++                                if(!net.minecraftforge.event.ForgeEventFactory.onLivingUpdateBlock(this, blockpos, Blocks.field_150350_a.func_176223_P()))
++                                    flag1 = this.field_70170_p.func_175698_g(blockpos) || flag1;
```

Comment:
what? 
I'm literally saying the patch would go from -1, +2, to just +1 if you didn't make the flag1 line indented
Java doesn't care about indentation